src/views/Records/NetworkGraph.vue

```diff
@@ -0,0 +1,272 @@
+                    },
+                    plotOptions: {
+                        networkgraph: {
+                            keys: ['from', 'to', 'rel'],
+                            layoutAlgorithm: {
+                                enableSimulation: true,
+                                linkLength: 60,
+                                maxIterations: 120
+                            }
+                        },
+                    },
+                    series: [{
+                        animation: false,
+                        events: {
+                          click: /* istanbul ignore next */ async function(event) {
+                            // Avoid redundant navigation to self...
+                            if (parseInt(_module.$route.params.id) !== parseInt(event.point.record_id)) {
+                              _module.$router.push({
+                                path: "/graph/" +  event.point.record_id
+                              })
+                            }
+                          }
+                        },
+                        dataLabels: {
+                            enabled: true,
+                            linkFormat: '{point.rel}',
+                            color: '#2F2F30',
+                            font: 'light 30px "Trebuchet MS", Verdana, sans-serif',
+                            linkTextPath: {
+                              attributes: {
+                                dy: 12
+                              }
+                            },
+                        formatter: /* istanbul ignore next */ function() {
+                            // this is not the component but the point
+                            if (this.key.length < 40) {
+                              return this.key
+                            }
+                            return this.key.substring(0, 40) + "..."
+                          }
+                        },
+                        link: {
+                          color: "rgba(0, 0, 0, 0.5)",
+                          width: 2,
+                        },
+                        id: 'relationships',
+                        data: [],
+                        states: {
+                          inactive: {
+                            enabled: true,
+                            linkOpacity: 0.000001,
+                            opacity: 0.000001
+                          }
+                        }
+                    }],
+                    nodes: null
+                },
+                relations: null
+            }
+        },
+        computed: {
+          currentRoute() {
+            return this.target || this.$route.params['id'];
+          }
+        },
+        watch: {
+          async currentRoute() {
+            await this.getData();
+          },
+          async max_path_length() {
+            await this.getData();
+          }
+        },
+        mounted() {
+            this.$nextTick(async function () {
```

Comment:
can be simplified to this although it is correct already. 

![image](https://user-images.githubusercontent.com/25683981/114159374-a1a55380-991d-11eb-9676-61d1ba638eda.png)

Reply:
It's fine to use async mounted. AS a warning though, I've encountered issues in some unit tests where I had to force a next tick to get the method to be correctly executed.